productivity-suite/app/legacy-app/src/components/pages/News.tsx

```diff
@@ -11,9 +16,57 @@ export function News() {
     };
   });
 
+  useEffect(() => {
+    if (currentPageRef.current === undefined) {
+      const pageDetails = getBus().latestValue<{ page: number }>(
+        "update-news-page-number"
```

Comment:
`update-new-page-number` and `update-news-page-view` seem very similar to me and it is difficult to tell which does what. Could we make them less similar?

Could it be simply: `news-page` and `change-news-page`?